fineract-provider/src/main/resources/sql/migrations/core_db/V7001__GLIM_migration_script.sql

```diff
@@ -0,0 +1,58 @@
+--
```

Comment:
Kindly use one migration script (V344__GSIM_migration_script.sql) or if you feel 2 migration files are required, use the next available sequence number